src/App.tsx

```diff
@@ -33,8 +34,22 @@ const App: React.FC = () => {
               <Switch>
                <Route exact path="/"
```

Comment:
Special handling for root ("/") path.  On successful login, go directly to the redirectURL.